src/scss/base/_typography.scss

```diff
@@ -5,30 +5,26 @@
 @use '../settings/include-media' as *;
 
 @mixin typography() {
-    /**
-    * Typography
-    * =================================
-    * - Base
-    * - Paragraphs
-    * - Headings
-    * - Type sizes
-    * - Misc
-    * - Utilities
-    *
-    * Body font size, leadings etc have been set in _variables.scss
-    *
-    * This is an optional component within Fozzie.
-    * If you'd like to use it in your project you can include it by adding `@include typography();` into your SCSS dependencies file.
-    *
-    * Documentation:
-    * TBC
-    */
-
-
-    /**
-    * Typography preloading styles
-    */
+    // **
```

Comment:
Any of these inside a mixin are fine to be loud comments as they'll be output in context in the CSS